politeiawww/cmd/politeiawwwcli/commands/help.go

```diff
@@ -14,5 +14,21 @@ func (cmd *HelpCmd) Execute(args []string) error {
 		fmt.Printf("%s\n", LoginCmdHelpMsg)
 	}
 
+	if cmd.Args.Topic == "logout" {
```

Comment:
There are going to be quite a few commands so I think this would read better if it were a switch statement.  Example:

```
switch cmd.Args.Topic {
  case "logout":
    fmt.Printf("%s\n", LogoutCmdHelpMsg)
  case "authorizevote":
    fmt.Printf("%s\n", AuthorizeVoteCmdHelpMsg)
  default: 
    fmt.Printf("invalid command\n")
}
```

Reply:
Done